FatturaElettronica.csproj

```diff
@@ -1,5 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup Label="Globals">
```

Comment:
Questo blocco andrebbe rimosso in quanto non utile al progetto.

Reply:
blocco rimosso, grazie